packages/slate-hotkeys/src/index.js

```diff
@@ -87,6 +106,7 @@ const isContentEditable = e =>
   isDeleteLineForward(e) ||
   isDeleteWordBackward(e) ||
   isDeleteWordForward(e) ||
+  isOtherDelete(e) ||
```

Comment:
Instead of adding `isOtherDelete` here, can we just re-use the existing `isBackspace` or `isDelete` from above? That seems like it would solve it in a way that's more clear and with less code.

Reply:
@ianstormtaylor Yes, we can.  I have put `e.key === ` here.   Fixed in a53e9918